src/server/multi_command_squasher.cc

```diff
@@ -63,6 +66,10 @@ size_t Size(const facade::CapturingReplyBuilder::Payload& payload) {
 }  // namespace
 
 atomic_uint64_t MultiCommandSquasher::current_reply_size_ = 0;
+thread_local size_t MultiCommandSquasher::throttle_size_limit_ =
+    absl::GetFlag(FLAGS_throttle_squashed);
```

Comment:
As discussed this morning multiply by thread count. The limit should be per thread and the current_reply_size_ is global counter

Reply:
Yes I know, I even wrote a comment above that I will follow up with this :smile: 

I wanted to know if you have anything else to add :smile: